### PR TITLE
Add an Ant task

### PR DESCRIPTION
**Pull request by ricksbrown**
Hi there - I need to invoke `sass-java` from Ant and have written a custom task.

I thought you might like this as part of the core project - a Maven plugin would also make sense at some point and an Ant task is already much of the Mojo work done.

If you would prefer not to merge this PR please let me know and I'll publish this task separately.

### CODE REVIEW

**Comment by headcr4sh** [comment]
Hi Rick.
Thanks for the PR.

Ant task and a Maven plugin seem to be the next things that need to be done to leverage the power of the sass binding.

I'll perform a review as soon as possible (might be next week, though) and if I don't have any concerns about your code, I'll gladly merge your PR.

I am thinking about splitting up node-sass into various modules (sassj, libsass, ant-task, maven-plugin, spring-integration, cdi-component,...), so I might end up re-structuring the whole project sooner or later at this point.

PS:
Well, If found the time to perform a review today... surprise.. ;-)
See comment below for further details.

**Comment by headcr4sh** [comment]
@ricksbrown I have a question and a request concerning your pull request.
Once the two topics below have been clarified / implemented, I'll gladly merge your stuff into the "master" branch of "sass-java" and prepare a new release afterwards.

1) QUESTION
Is it intended that the attributes of the Ant task aren't written CamelCase-style? I just had a look at some of the "official" Ant task and was a bit surprised that they seem to sport a massive mix-up of alllowercaseattributes and camelCaseAttributes and I couldn't really figure out what would be best practice for new Ant tasks.

2) REQUEST
Please add yourself to the "contributors" section of pom.xml. (unless you don't want to be mentioned,..)
I would really like to give proper credits to all code contributors.

File: src/main/java/com/cathive/sass/SassTask.java [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    private Boolean sourceComments = null;
+    private Boolean sourceMapEmbed = null;
+    private Boolean sourceMapContents = null;
+    private Boolean omitSourceMapUrl = null;
+    private Boolean isIndentedSyntaxSrc = null;
+    private String sourceMapFile = null;
+    private String sourceMapRoot = null;
+    private File ouputPath = null;
+    private File in = null;
+    private final Collection<org.apache.tools.ant.types.Path> paths = new ArrayList<org.apache.tools.ant.types.Path>();
```
Comment (headcr4sh) [inline src/main/java/com/cathive/sass/SassTask.java]: The source code level is 1.7, so we can use the "diamond operator".

File: src/main/java/com/cathive/sass/SassTask.java [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    private Integer precision = null;
+    private SassOutputStyle outputStyle = null;
+    private Boolean sourceComments = null;
+    private Boolean sourceMapEmbed = null;
+    private Boolean sourceMapContents = null;
+    private Boolean omitSourceMapUrl = null;
+    private Boolean isIndentedSyntaxSrc = null;
+    private String sourceMapFile = null;
+    private String sourceMapRoot = null;
+    private File ouputPath = null;
```
Comment (headcr4sh) [inline src/main/java/com/cathive/sass/SassTask.java]: Typo! s/ouputPath/outputPath/

File: README.md [hunk 3]
```diff
@@ -83,4 +83,74 @@ class SimpleSassExample {
...
+    <dependencies>
+        <dependency>
+            <groupId>com.cathive.sass</groupId>
+            <artifactId>sass-java</artifactId>
+            <version>${sass-java.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.3</version>
```
Comment (headcr4sh) [inline README.md]: Ant v1.9.6 is the current version. If there is no particular reason to stick with v1.9.3, I would reference the latest release.

File: pom.xml [hunk 4]
```diff
@@ -249,5 +249,17 @@
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.2</version>
```
Comment (headcr4sh) [inline pom.xml]: Ant v1.9.6 is the current version. If there is no particular reason to stick with v1.9.3, I would reference the latest release.

File: pom.xml [hunk 5]
```diff
@@ -249,5 +249,17 @@
...
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant-testutil</artifactId>
+            <version>1.9.2</version>
```
Comment (headcr4sh) [inline pom.xml]: Ant v1.9.6 is the current version. If there is no particular reason to stick with v1.9.3, I would reference the latest release.

**Comment by ricksbrown** [comment]
@headcr4sh Thanks for the speedy review, much appreciated.

FYI when I changed Ant version from `1.9.2` to `1.9.6` the unit test class I had used was now deprecated so I had to refactor it to use the new class.

To answer your question on Ant attributes: you do sometimes see camelcase attributes on Ant tasks but it is rare. I cannot think of any core Ant tasks that use camelcase even when the attribute name is quite long, for example: `ignoresystemclasses` on the [available](https://ant.apache.org/manual/Tasks/available.html) task.

Edit: the Ant source code for that attribute is:

``` java
public void setIgnoresystemclasses(boolean ignore) {
    this.ignoreSystemclasses = ignore;
}
```

The [Writing Your Own Task](https://ant.apache.org/manual/develop.html) tutorial says:

> The name of the method must begin with set, followed by the attribute name, with the first character of the name in uppercase, and the rest in lowercase*.
> *Actually the case of the letters after the first one doesn't really matter to Ant, using all lower case is a good convention, though.

**Pull request merged**